### ISSUE TITLE
Support ?format=kubernetes for ServiceInsight resources

### ISSUE THREAD

**Issue by kleinfreund** [kumahq/kuma#6780]
### Description

TL;DR: Add support for `?format=kubernetes` on the `/meshes/${mesh}/service-insights/${name}` endpoint.

- In https://github.com/kumahq/kuma/pull/6673, support for the `format` query parameter was added to all resource endpoints
- In https://github.com/kumahq/kuma-gui/issues/696, we’re making use of that feature to provide a "Copy as Kubernetes" button for code blocks showing resources which (1) queries the resource in question with `?format=kubernetes` and (2) copies it.
- To add this feature everywhere in the UI we show resources, we need this feature for `ServiceInsight` resources which is currently not supported.

The following endpoints are used in the UI for this purpose:

- `/meshes/${name}`
- `/meshes/${mesh}/dataplanes/${name}`
- `/meshes/${mesh}/service-insights/${name}`
- `/meshes/${mesh}/external-services/${name}`
- `/meshes/${mesh}/${path}/${name}`

**Comment by lahabana** [comment]
I've actually went and looked at this and I'm not sure the k8s format makes any sense here. There's no CRD for service-insights they are just derived from the larger entity.
Moving to triage to discuss not doing this at all.

**Comment by kleinfreund** [comment]
@lahabana Should we, for the time being, hide the "copy as k8s" button for service insights in UI then?

**Comment by lahabana** [comment]
Yes in the interim I think it's the right thing to do!
